.circleci/config.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           docker-compose
           up
           --build
-          --exit-code-from test
+          --abort-on-container-exit
```

Comment:
i had something cause `web` to exit before `test` could be started and it returned 0 because test wasn't run >_<. this flag is a subset of `--exit-code-from=<container>` that appears to use the exit code of the container that caused the abort, which is what we want.